### ISSUE TITLE
Geofence plugin methods not returning any response (no success or failure callbacks)

### ISSUE THREAD

**Issue by kanikaluthra14** [transistorsoft/cordova-background-geolocation-lt#943]
We are using the cordova-background-geolocation-lt plugin in our cordova app to update user status whenever he enters or exits the periphery location. For this, we utilized BackgroundGeolocation methods addGeofence, onGeofence, ready, etc.  We were able to successfully achieve the functionality and tested the same. 

Problem occurred when we created a fresh git project and installed this plugin again. On debugging I found that plugin methods are getting called successfully, but their successcallback and failurecallback are not getting returning.

## Environment
* Plugin version:
cordova-background-geolocation-lt ="^2.14.2"
              GOOGLE_API_VERSION value="11.8.0"
              LICENSE ="YOUR_LICENSE_KEY" 
              APPCOMPAT_VERSION="27.1.1"
* Platform: iOS or Android - Android
* OS version: Windows 10 64 bit OS
* Device manufacturer / model: Dell
* Cordova version (`cordova -v`): 8.1.2
* Cordova platform version (`cordova platform ls`):
Installed platforms:
  android 7.1.4
Available platforms:
  browser ~5.0.1
  ios ~4.5.4
  osx ~4.0.1
  windows ~6.0.0
* Plugin config provided to `#ready`: Yes

```//Configure Geofence
    var configureGeofence = function () {
        alert('configure geofence called' + BackgroundGeolocation ? 'Found' : 'Not found');
        BackgroundGeolocation.ready({
            reset: true,
            desiredAccuracy: BackgroundGeolocation.DESIRED_ACCURACY_HIGH,
            distanceFilter: 10,
            geofenceInitialTriggerEntry: true,
            url: 'http://dummyurl.com',
            autoSync: true,
            desiredAccuracy: 0,
            stationaryRadius: 1,
            distanceFilter: 20,
            debug: false,
            stopOnTerminate: false,
            activityType: 'AutomotiveNavigation',
            notificationTitle: 'Background tracking', // <-- android only, customize the title of the notification
            notificationText: 'ENABLED',
            useSignificantChangesOnly: false,
            locationAuthorizationRequest: 'Always',
            activityRecognitionInterval: 100,
            heartbeatInterval: 60,
            preventSuspend: true,
            geofenceProximityRadius: 1000,
            locationUpdateInterval: 3000,
            foregroundService: true,
            allowsBackgroundLocationUpdates: true,
            schedule: [
                '1-7 9:00-22:00',   // Mon-Fri: 9:00am to 10:00pm
            ],
            headers: {
                "Authorization":'token'
            },
            params: {
                'Data': '1234',
            }
        }, state => {
            alert('Starting geofencing');
            console.log('Starting geofencing', state);
            BackgroundGeolocation.startGeofences();
        });
    };```

## Expected Behavior
On registering plugin ready method and add method, successcallback should return a valid response and alert should pop up as shown in the code above.

## Actual Behavior
On registering plugin ready method and add method, neither successcallback nor failurecallback is getting returned, so we are not getting any response.

## Context
I was trying to invoke removeGeofence method, and on its success I was adding a new geofence. Furthermore, I configured Geofence using ready method followed by registering its listener using onGeofence method.

Also note, that I did not use removeGeofence method before, so is there a possibility that the number of Geofences that can be registered on a device  has exceeded?

<details>
	<summary>Logs</summary>
[Geofence Logs.txt](https://github.com/transistorsoft/cordova-background-geolocation-lt/files/3308963/Geofence.Logs.txt)

```<!-- Syntax highlighting:  DO NOT REMOVE -->
PASTE_YOUR_LOGS_HERE
```
</details>

**Comment by christocracy** [comment]
Show me `$ adb logcat` of your app booting.

**Comment by kanikaluthra14** [comment]
[Geofence Logs.txt](https://github.com/transistorsoft/cordova-background-geolocation-lt/files/3313447/Geofence.Logs.txt)

**Comment by christocracy** [comment]
@kanikaluthra14 Your problem is too low a version for `GOOGLE_API_VERSION: 11.8.0`:
```
06-20 11:16:29.329 11141 11371 W System.err: java.lang.NoClassDefFoundError: com.google.android.gms.location.ActivityTransitionEvent
```
Configure it to latest available `16.0.0`
